dawn/src/dawn/Optimizer/PassTemporaryToStencilFunction.cpp

```diff
@@ -767,6 +769,6 @@ bool PassTemporaryToStencilFunction::run(
   }
 
   return true;
-}
+} // namespace dawn
```

Comment:
this was probably due to a parenthesis mismatch at some point, should not be here